From 850208dde266370c0143a9fb206c283aa30a85e3 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Wed, 16 Nov 2022 18:12:51 +0100
Subject: [PATCH 233/954] feat(fdts): add eMMC node on stm32mp257f-ev

Fill sdmmc2 node which is used for eMMC on stm32mp257f-ev board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 15434b3f07b2d8fde739fbbbf4082ae0a4aa435c feat(fdts): add eMMC node on stm32mp257f-ev]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I6b1a200697c86c33d31cd7ce09fc27284f080bca
---
 fdts/stm32mp257f-ev1.dts | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/fdts/stm32mp257f-ev1.dts b/fdts/stm32mp257f-ev1.dts
index 390510cff6..bc2cc79eec 100644
--- a/fdts/stm32mp257f-ev1.dts
+++ b/fdts/stm32mp257f-ev1.dts
@@ -54,6 +54,17 @@
 	status = "okay";
 };
 
+&sdmmc2 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&sdmmc2_b4_pins_a &sdmmc2_d47_pins_a>;
+	non-removable;
+	no-sd;
+	no-sdio;
+	st,neg-edge;
+	bus-width = <8>;
+	status = "okay";
+};
+
 &usart2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&usart2_pins_a>;
-- 
2.53.0

